src/cargo/util/process_builder.rs

```diff
@@ -384,4 +453,9 @@ mod imp {
         // Just execute the process as normal.
         process_builder.exec()
     }
+
+    pub fn command_line_too_big(err: &std::io::Error) -> bool {
+        const ERROR_FILENAME_EXCED_RANGE: i32 = 206;
```

Comment:
Since this was initially written into rustc this now exists in `winapi`, so could the constant there be used?